runtime/help/options.md

```diff
@@ -462,6 +462,17 @@ Here are the available options:
 
     default value: `false`
 
+* `truecolor`: controls the usage of the true color support/mode.
```

Comment:
What is "support/mode"? Also, English is not my native language but I think "the" here is completely redundant, and it takes up space.

So, again, why not make it brief yet clear and exhaustive, e.g.: "controls whether micro will use true colors (24-bit colors) when using a colorscheme with true colors, such as `solarized-tc` or `atom-dark`".